.travis.yml

```diff
@@ -1,5 +1,5 @@
 dist: trusty
-
+sudo: required
```

Comment:
Unless we really need sudo, we should not use it. Builds with sudo takes a significant longer time to start.